src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -387,73 +438,210 @@ public void testTestPermissions() {
-            .build()
-            .getService();
-    EasyMock.expect(resourceManagerRpcMock.get(PARTIAL_PROJECT.getProjectId(), EMPTY_RPC_OPTIONS))
-        .andThrow(
-            new ResourceManagerException(
-                403, "Project " + PARTIAL_PROJECT.getProjectId() + " not found."))
-        .once();
-    EasyMock.replay(resourceManagerRpcMock);
+    String exceptionMessage = "Project " + PARTIAL_PROJECT.getProjectId() + " not found.";
+    Mockito.doThrow(new ResourceManagerException(403, exceptionMessage))
+        .when(resourceManagerMock)
+        .get(PARTIAL_PROJECT.getProjectId());
     try {
       resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
       fail();
-    } catch (ResourceManagerException e) {
-      assertTrue(e.getMessage().contains("Project partial-project not found"));
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
     }
   }
 
   @Test
   public void testRuntimeException() {
-    ResourceManagerRpcFactory rpcFactoryMock = EasyMock.createMock(ResourceManagerRpcFactory.class);
-    ResourceManagerRpc resourceManagerRpcMock = EasyMock.createMock(ResourceManagerRpc.class);
-    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(ResourceManagerOptions.class)))
-        .andReturn(resourceManagerRpcMock);
-    EasyMock.replay(rpcFactoryMock);
-    ResourceManager resourceManagerMock =
-        ResourceManagerOptions.newBuilder()
-            .setServiceRpcFactory(rpcFactoryMock)
-            .build()
-            .getService();
     String exceptionMessage = "Artificial runtime exception";
-    EasyMock.expect(resourceManagerRpcMock.get(PARTIAL_PROJECT.getProjectId(), EMPTY_RPC_OPTIONS))
-        .andThrow(new RuntimeException(exceptionMessage));
-    EasyMock.replay(resourceManagerRpcMock);
+    Mockito.doThrow(new RuntimeException(exceptionMessage))
+        .when(resourceManagerMock)
+        .get(PARTIAL_PROJECT.getProjectId());
     try {
       resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
       fail();
-    } catch (ResourceManagerException exception) {
-      assertEquals(exceptionMessage, exception.getCause().getMessage());
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testClearOrgPolicy() {
+    String exceptionMessage = "Should fail because the organization policy doesn't exist.";
+    Mockito.doThrow(new ResourceManagerException(404, exceptionMessage))
+        .when(resourceManagerMock)
+        .clearOrgPolicy(RESOURCE, ORG_POLICY_INFO);
+    try {
+      resourceManagerMock.clearOrgPolicy(RESOURCE, ORG_POLICY_INFO);
+      fail();
+    } catch (RuntimeException expected) {
+      assertEquals(exceptionMessage, expected.getMessage());
+    }
+  }
+
+  @Test
+  public void testGetEffectiveOrgPolicy() {
+    Mockito.when(resourceManagerMock.getEffectiveOrgPolicy(RESOURCE, CONSTRAINTS))
```

Comment:
You've mocked out the method you're trying to test.

Reply:
done